.github/workflows/e2e_test_ci.yml

```diff
@@ -472,6 +474,43 @@ jobs:
           if [[ -n "$ENGINE" ]]; then
             echo "    acceleration:" >> spicepod.yaml
             echo "      enabled: true" >> spicepod.yaml
+            echo "      refresh_sql: SELECT * FROM test_postgresql_table WHERE int2_column is NULL or int2_column != 10" >> spicepod.yaml
```

Comment:
Learned today that `int2_column != 10` will also filter out records where value is NULL (native engines behavior)